src/mame/act/victor9k.cpp

```diff
@@ -679,28 +713,32 @@ void victor9k_state::machine_reset()
 void victor9k_state::victor9k(machine_config &config)
 {
 	// basic machine hardware
-	I8088(config, m_maincpu, XTAL(30'000'000)/6);
+	LOGDISPLAY("start of configure\n");
```

Comment:
Please don’t put logging in machine configuration functions.  It’s going to mess stuff up.

Reply:
Thanks for the feedback. I removed this.